loader/trampoline.c

```diff
@@ -468,7 +435,7 @@ LOADER_EXPORT VKAPI_ATTR VkResult VKAPI_CALL vkCreateInstance(const VkInstanceCr
     ptr_instance->magic = LOADER_MAGIC_NUMBER;
 
     // Save the application version
-    if (NULL == pCreateInfo || NULL == pCreateInfo->pApplicationInfo || 0 == pCreateInfo->pApplicationInfo->apiVersion) {
+    if (NULL == pCreateInfo->pApplicationInfo || 0 == pCreateInfo->pApplicationInfo->apiVersion) {
```

Comment:
Is this removing a nullptr check? Seems unsafe to remove that unless it is being checked before and thus guaranteed to not be null.

Reply:
Won't get here.  Lines 400-404 do this check and return.